diff --git a/gui/Android.mk b/gui/Android.mk
index 9a5f36c1..755b6437 100755
--- a/gui/Android.mk
+++ b/gui/Android.mk
@@ -230,14 +230,14 @@ ifeq ($(BOARD_RECOVERYIMAGE_PARTITION_SIZE),)
     endif
     REQUIRED_RECOVERY_SIZE := $(BOARD_BOOTIMAGE_PARTITION_SIZE)
 endif
-ifeq ($(shell test $(REQUIRED_RECOVERY_SIZE) -le 21073920; echo $$?),0)
+ifeq ($(shell test $(REQUIRED_RECOVERY_SIZE) -le 25165824; echo $$?),0)
     $(warning Recovery Size is small discarding Special fonts)
 endif
 TWRP_RES_GEN := $(intermediates)/twrp
 $(TWRP_RES_GEN):
 	mkdir -p $(TARGET_RECOVERY_ROOT_OUT)$(TWRES_PATH)
 	cp -fr $(TWRP_RES) $(TARGET_RECOVERY_ROOT_OUT)$(TWRES_PATH)
-	if test $(REQUIRED_RECOVERY_SIZE) -le 21073920; then \
+	if test $(REQUIRED_RECOVERY_SIZE) -le 25165824; then \
 	    rm -rf $(TARGET_RECOVERY_ROOT_OUT)$(TWRES_PATH)/fonts/DroidSansFallback.ttf; \
 	    rm -rf $(TARGET_RECOVERY_ROOT_OUT)$(TWRES_PATH)/fonts/NotoSansCJKjp-Regular.ttf; \
 	    rm -rf $(TARGET_RECOVERY_ROOT_OUT)$(TWRES_PATH)/fonts/RoboNoto-Medium.ttf; \
